### ISSUE TITLE
UART over BLE

### ISSUE THREAD

**Issue by clemaitre58** [kriswiner/nRF51822_IoT_Breakout#4]
@kriswiner 

Do you have some example that show how to use your board with the uart over BLE that Nordic provide in is SDK and example?

Thanks,
Cédric

**Comment by kriswiner** [comment]
Which board are talking about?

**Comment by clemaitre58** [comment]
@kriswiner 

this one : https://www.tindie.com/products/onehorse/nrf52832-development-board/

**Comment by kriswiner** [comment]
You can program this using the Arduino core from Sandeep Mistry and the
J-link from an nRF52 development kit. I am not sure I understand your
question. This is a standalone microcontroller. There are examples on
github for using the BLE radio embedded in the chip to send sensor data to
a smartphone. is this what you mean?

**Comment by clemaitre58** [comment]
@kriswiner 

You could create a service with which is a UART over BLE : https://developer.nordicsemi.com/nRF5_SDK/nRF51_SDK_v8.x.x/doc/8.0.0/s110/html/a00072.html

In this manner, you could push on BLE fron NRF52 a buffer of const char *.
This exactly the same things from computer or smartphone.

My question is : do you some example of implementation of this service?

TY
Cédric

**Comment by kriswiner** [comment]
Yes, there is the standard Nordic UART service. Yiou can find an example
here.

https://github.com/sandeepmistry/arduino-BLEPeripheral/blob/master/examples/serial/serial.ino

**Comment by clemaitre58** [comment]
:+1:
